apps/dashboard/components/agents/agent-details.tsx

```
@@ -22,6 +33,35 @@
+  const calculateResourceSize = (resource: TypedResource<ResourceType>) => {
+    switch (resource.type) {
+      case 'contract':
+        const value = resource.value as ContractValue;
+        return countBytes(
+          value.abi +
+            value.address +
+            (value.abi || CONTRACT_EXTRA_SIZE_COST) +
+            CONTRACT_EXTRA_SIZE_COST
+        );
+      case 'link':
+      case 'link': {
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)